tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -94,6 +94,32 @@ describe('Lobby - Page Content (Ranked)', () => {
     checkRanked(true);
     cy.get('[data-cy=ready-button-sword-cross-icon]').should('exist');
   });
+
+  it('Toggles ranked/casual switch with "Enter" key', () => {
+    // Focus the ranked/casual switch
+    cy.get('[data-cy=edit-game-ranked-switch]').focus();
+
+    // Check the initial state (ranked mode)
+    checkRanked(true);
+    cy.get('[data-cy=ready-button-sword-cross-icon]').should('exist');
+
+    // Simulate pressing the 'Enter' key
+    cy.get('[data-cy=edit-game-ranked-switch]').type('{enter}');
```

Comment:
Same idea for all the references 
```suggestion
    cy.get('[data-cy=edit-game-ranked-switch] input').type('{enter}');
```